astropy/io/votable/tree.py

```diff
@@ -25,7 +25,7 @@
 from ...utils.xml.writer import XMLWriter
 from ...utils.exceptions import AstropyDeprecationWarning
 from ...utils.misc import InheritDocstrings
-
+from ...units import Quantity
```

Comment:
Is `Quantity` actually used below? More generally, are any of the changes here needed for the PR? I got a bit confused...